src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+    }
+    public static void show_list(int num){
+        System.out.println(" ____________________________________________________________\n" +
+                "     Here are the tasks in your list:");
+        for(int i=0; i< num;i++){
+            System.out.println((i+1) +". "+ "["+t[i].getSymbol()+"]"+"[" + t[i].getStatusIcon()+"] "+ t[i].getDescription());
+        }
+    }
+
+    public static String userDone(String user_done){
+        switch(user_done){
+            case "todo":
+                return "T";
+            case "event":
+                return "E";
+            case "deadline":
+                return "D";
+            default:
+                break;
+        }
+        return "Wrong input";
+
+
+    }
+    public static void printTask(String taskKind, String substring,int num_of_goals, String ddl,String prep){
+        System.out.println("Got it. I've added this task: ");
+        if(ddl == null){
+            System.out.println("[" + taskKind + "]" + "[" + " " + "]" + " " + substring);
+        }
+        else{
+            System.out.println("[" + taskKind + "]" + "[" + " " + "]" + " " + substring + "(" + prep +":"+ddl +")");
+        }
+
+        System.out.println("Now you have " + num_of_goals + " task in your list!");
+    }
+
+
+    public static void main(String[] args) {
+
+        show_welcome_msg();
+        Scanner sc = new Scanner(System.in);
+        String user_input = sc.nextLine();
+        int num_of_goals = 0;
+
+        while(!user_input.equals("bye")) {
+            if (user_input.equals("list")) {
+                show_list(num_of_goals);
+            }
+
+            else {
+                int spacing = user_input.indexOf(" ");
+                int slash = user_input.indexOf("/");
+                System.out.println(slash);
+                String user_done = user_input.substring(0, spacing);
+                if (user_done.equals("done")){
+                    String number = user_input.substring(spacing+1);
+                    int number_converted = Integer.parseInt(number);
+                    t[number_converted-1].markAsDone();
+                    System.out.println("____________________________________________________________\n" +
+                            "     Nice! I've marked this task as done: " );
+                    System.out.println("["+t[number_converted-1].getSymbol()+"]"+"[" + t[number_converted-1].getStatusIcon()+"] "+ t[number_converted-1].getDescription());
```

Comment:
Can be confusing due to long expression. Takes reader time to decipher what it means.